mobile-app/app/screens/AppNavigator/screens/Loans/screens/EditCollateralScreen.tsx

```diff
@@ -225,7 +226,7 @@ export function EditCollateralScreen ({
                   onCloseButtonPress: dismissModal,
                   navigateToScreen: {
                     screenName: 'AddOrRemoveCollateralForm',
-                    onButtonPress: onAddCollateral
+                    onButtonPress: onAddCollateral as any
```

Comment:
Why do we need this?

Reply:
because I get this error:
229:36  error    Promise-returning function provided to property where a void return was expected                                                    @typescript-eslint/no-misused-promises